packages/frontend-dynamic-feature-loader/report.api.md

```diff
@@ -0,0 +1,18 @@
+## API Report File for "@backstage/frontend-dynamic-feature-loader"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { CreateAppFeatureLoader } from '@backstage/frontend-defaults';
+import { init } from '@module-federation/enhanced/runtime';
+
+// @public
+export function dynamicFrontendFeaturesLoader(
+  options?: DynamicFrontendFeaturesLoaderOptions,
+): CreateAppFeatureLoader;
+
+// @public (undocumented)
+export type DynamicFrontendFeaturesLoaderOptions = {
+  moduleFederation: Omit<Parameters<typeof init>[0], 'name' | 'remotes'>;
```

Comment:
I wonder if this is risky (passthrough of a type). Even small bumps of the module-federation library might bring breaking changes for users of this function now right? Maybe it's OK, no strong opinion, but we can't "shield" end users from changing underlying interfaces now, especially in the case of parameters where we would have preferred to provide a default or where usage of the parameter actually can lead to breakages in our implementation. Any opinions there?

Reply:
I changed this in commit 7452c175b70c1df851251dfa7adc582d529bb5de to use publicly exported Module Federation types.